### PR TITLE
firebarrels data and visual improvements

### PR DESCRIPTION
**Pull request by meiciuc**

### CODE REVIEW

**Comment by YarRainbow** [comment]
@mike-lvov it's in testing yet, don't merge. I inform you later.

File: firestore.rules [hunk 1]
```diff
@@ -65,7 +65,8 @@ service cloud.firestore {
       }
       match /firebarrels/{firebarrelId} {
         allow read: if request.auth.uid != null;
-        allow write: if false;
+        allow create, update: if (request.resource.data.diff(resource.data).affectedKeys().hasOnly(['connectedUsers']));
+        allow delete: if false;
```
Comment (mike-lvov) [inline firestore.rules:69]: What's this for?

Comment (mike-lvov) [reply firestore.rules:69]: add the auth check + debt comment that any user can set an empty arr, please

**Review by mike-lvov — APPROVED** [review]
LGTM

**Pull request merged**